CHANGELOG.md

```diff
@@ -4,73 +4,66 @@ All notable changes to this project will be documented in this file. See [commit
 
 ## [5.4.0](https://github.com/wagoid/commitlint-github-action/compare/v5.3.1...v5.4.0) (2023-04-10)
 
-
```

Comment:
Let's rollback these changes, as this file is automatically generated 💪

Reply:
👍

Reply:
do i need to do anything for this in this PR?

Reply:
Nop, it gets automatically generated from your commit messages 👍